src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,57 @@
 
 #include "t06_homework.h"
 #include <iostream>
-
+#include<string>
 using namespace std;
+string low(string);
 
+bool IsHavingOneBigSymbol(string word)
+{
+  int br=0;
+  for(int i=0;i<word.length();i++){
+    if( word[i] >= 'A' && word[i] <= 'Z'){br++;}
+  }
+  return (br==1);
+}
+bool IsInTheDicAndCorrect(const string word, string* dic,int n)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'word' should be passed by const reference.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4586365)